packages/concerto-types/scripts/codegen.js

```diff
@@ -4,12 +4,14 @@ const { ModelManager } = require('@accordproject/concerto-core');
 const { CodeGen: { TypescriptVisitor }} = require('@accordproject/concerto-codegen');
 const { FileWriter } = require('@accordproject/concerto-util');
 const path = require('path');
+const {DCS_MODEL }= require('@accordproject/concerto-core/lib/decoratormanager');
```

Comment:
@mttrbrts is there any better way to import the DCS model?

Reply:
Should we just hard code the model here in `concerto-types`?

Reply:
I think that this is reasonable for now given that this is the home for the DCS model. However, I would align better with the metamodel to move the model into the @accordproject/concerto-metamodel package, and import from there.

Let's open a separate task for that, given that it would be a breaking change (unless we do some kind of proxying to move it transparently).